01-resources-v2/commit-status.yml

```diff
@@ -0,0 +1,30 @@
+resource_types:
+- name: github-status
+  type: docker-image
+  source: {repository: concourse/github-status-resource}
+
+resources:
+- name: atc
+  type: git
+  source:
+    uri: https://github.com/concourse/atc
+  notify:
+  - on: build_started
+    using: atc-status # invoke atc-status notifier with 'self' set to this resource
+  - on: build_finished
+    using: atc-status # invoke atc-status notifier with 'self' set to this resource
+
+- name: atc-status
+  type: github-status
+  source:
+    repository: concourse/atc
+    access_token: ((token))
+
+jobs:
+- name: atc-pr-unit
+  plan:
+  - get: atc-pr
+    trigger: true
+    spaces: all
```

Comment:
What happens to `versions: every`? With spaces, would it be possible to build every single commit on all open pull requests by setting `spaces: all` and `versions: every`?

Reply:
Yep, that's how the two would interact.